gradle/libs.versions.toml

```diff
@@ -20,7 +20,6 @@ buildconfig = "5.5.1"
 
 [libraries]
 junit-jupiter = { module = "org.junit.jupiter:junit-jupiter", version.ref = "junit-jupiter" }
-junit-platform-launcher = { module = "org.junit.platform:junit-platform-launcher" }
```

Comment:
we don't need the above for `junit-jupiter` is enough. See example usage of JUnit dependency in gradle at https://github.com/junit-team/junit5-samples/blob/r5.4.0/junit5-jupiter-starter-gradle-kotlin/build.gradle.kts

Reply:
It is not 100% correct for Gradle v9 will require explicit testing platform spec